### ISSUE TITLE
fx140: aria labels of menulists are ignored by voiceover

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#5424]
https://bugzilla.mozilla.org/show_bug.cgi?id=1975696

For example, in `itemBox`, when item type `menulist` is focused, voiceover announces the current type (e.g. "Book Section") but does not announce the actual label of the field "Item Type".

This is present in the current firefox esr140, per the discussion in the linked issue.

None of aria properties that I tried seem to help here, so we probably just have to wait for mozilla to fix it. It is really not great, since there are many `menulist`s in preferences, so this makes it way harder to navigate it with screen reader. The only good thing is that voiceover QuickNav can allow one to move the virtual cursor from the menulist onto the label and hear it that way.

**Comment by dstillman** [comment]
If this isn't fixed in ESR soon, we could probably use fx141 on macOS.

**Comment by abaevbog** [comment]
That linked issue is now closed but I don't think there is a new esr version yet.

**Comment by dstillman** [comment]
No, it's coming September 16.

**Comment by abaevbog** [comment]
When I try this now with voiceover, labels of `menulist`s unfortunately still don't get announced. My OS (and therefore voiceover) version is quite a bit behind, so I am updating it now to see if it will work better after the update.

I did discover that adding an explicit `role="combobox"` to `menulist`s helps voiceover to announce labels again, at least on my OS version. It doesn't really make much sense, since `menulist` already has an implicit `combobox` role (per the accessibility tab in dev tools) but it might be a good solution, in case if my old OS version is not the problem.

**Comment by dstillman** [comment]
Firefox 140.3.0esr candidate build is here, for comparison: https://ftp.mozilla.org/pub/firefox/candidates/140.3.0esr-candidates/build1/mac/en-US/Firefox%20140.3.0esr.dmg

**Comment by dstillman** [comment]
(Also, I was conflating this with macOS Tahoe fixes. There's no rush on this, since (as far as we can tell) there are no major issues in Tahoe that require rushing Z8.)

**Comment by abaevbog** [comment]
After updating my OS, the `menulist` labels were still not getting read out. I checked the 140.3.0esr build, and one of the few remaining `menulist`s directly in a chrome window has the same issue. The linked PR here has a "combobox" tweak that worked on my end with some more details on my `menulist` findings.

---

As a side note, this is one more instance of a11y issues that are specific to Chrome windows (vs content loaded in a `<browser>`).  That's in addition to numerous issues specific to JAWS. Most of Firefox content is already just `html` loaded in a `browser`, so I am a bit concerned that in the future there will be more and more issues like this popping up in chrome windows.
